tests/integration/test_render.py

```diff
@@ -194,3 +202,21 @@ def test_background_image(self):
         image_path = '/resource/drag-and-drop-v2/public/img/triangle.png'
         self.assertTrue(bg_image.get_attribute("src").endswith(image_path))
         self.assertEqual(bg_image.get_attribute("alt"), 'This describes the target image')
+
+    def test_borders_inactive(self):
+        self.load_scenario()
+        zones = self._get_zones()
+        for index, zone in enumerate(zones):
+            elt = 'zone-{}'.format(index + 1)
+            # Firefox does not provide a single "border-style", so we check "border-top-style"
+            self.assertEqual(self._get_border_style(elt, 'border-top-style'), 'none')
+            self.assertEqual(self._get_border_style(elt, 'border-top-width'), '0px')
```

Comment:
@agaylard To make this work with `_get_style`, pass the names of the JS attributes that corresponds to the CSS properties you want to check:

```python
self.assertEqual(self._get_style(zone_selector, 'borderTopStyle'), 'none')
self.assertEqual(self._get_style(zone_selector, 'borderTopWidth'), '0px')
```

(As you probably already guessed, there is a general rule underlying this: To translate names of CSS properties to the corresponding JS attributes, remove the dashes and camelCase individual components.)

> Firefox does not provide a single "border-style", so we check "border-top-style"

AFAIK it's not just Firefox that does this -- `border-style` is a shorthand for setting the styles of each of the four borders of an element to the same value, so browsers translate `border-style: dotted` into four separate properties (`border-{top,right,bottom,left}-style`). To make the tests more restrictive, it might make sense to iterate over a list of these properties and assert that each one of them has the correct value. (The same applies to `border-width` and `border-color`.)